internal/validator/handler.go

```diff
@@ -173,6 +174,9 @@ func (this *handler) validateContent(ctx context.Context, requestType string, da
 		}
 		if requestType == playbookSatPayloadHeaderValue {
 			err = validateWithSchema(ctx, this.schemas[1], true, line, events)
+			if e := validateSatHostUUID(line); e != nil {
```

Comment:
if the schema check fail why still trying to parse the events?

Reply:
Done! :+1: